modules/sdk/src/service/index.ts

```diff
@@ -514,10 +522,10 @@ export class ConnextSdk {
       }
 
       if (isRecipientAssetInput) {
-        senderAmountUi = formatUnits(senderAmountBn, this.senderChain?.assetDecimals!);
+        senderAmountUi = formatUnits(senderAmountBn.toString(), this.senderChain?.assetDecimals!);
```

Comment:
`formatUnits` should work with BN, you don't need to `toString()` it.

Reply:
i think it's new stuff.
BigNumberish doesn't allow BigNumber

Reply:
That's stupid, it's called `BigNumberish` lol. Ok then, proceed with this change.